drivers/modem/modem_cmd_handler.h

```diff
@@ -85,6 +100,9 @@ struct modem_cmd_handler_data {
 	/* locks */
 	struct k_sem sem_tx_lock;
 	struct k_sem sem_parse_lock;
+
+	/* return value from direct cmd handler */
+	int ret;
```

Comment:
Is this used?

Reply:
I mean: in the first commit, I see instances of where it's checked, but not assigned.

Reply:
Right, it is used in the wifi driver, ie next commit. Is it better to make the addition in the wifi commit?